tasks/configure-build.yaml

```diff
@@ -14,28 +17,41 @@ spec:
     - name: source
     - name: registry-auth
       optional: true
+  volumes:
+    - name: default-push-secret
+      csi:
+        readOnly: true
+        driver: csi.sharedresource.openshift.io
+        volumeAttributes:
+          sharedSecret: $(params.shared-secret)
   steps:
     - name: appstudio-configure-build
-      image: registry.access.redhat.com/ubi8-minimal@sha256:574f201d7ed185a9932c91cef5d397f5298dff9df08bc2ebb266c6d1e6284cd1
+      image: quay.io/redhat-appstudio/appstudio-utils:4580b3ba3012095ff3981e50b6bbf753d4afd4c3
+      volumeMounts:
+        - name: default-push-secret
+          mountPath: /secret/default-push-secret
       script: |
         #!/usr/bin/env bash
         echo "App Studio Configure Build"
 
+        DEF=/secret/default-push-secret/.dockerconfigjson
         AUTH=/workspace/registry-auth/.dockerconfigjson
         DEST=/workspace/source/.dockerconfigjson
-        echo "Looking for Registry Auth Config: $AUTH"
-        if [ -f "$AUTH" ]; then
+        echo "Looking for Registry Auth Config"
+        if [ -f "$AUTH" ] && [ -f "$DEF" ]; then
+          echo "$DEF and $AUTH found"
+          jq -M -s '.[0] * .[1]' $DEF $AUTH > $DEST
```

Comment:
slick ;-) .. if my refresher on `jq` help/man pages and stackoverflow (https://stackoverflow.com/questions/42011086/merge-arrays-of-json) are correct, this is combining the .dockerconfigjson's, where the `-s` is key, right?